view/task/edit.html

```diff
@@ -114,6 +114,24 @@ <h1 class="h2 crunch-top">Edit URL</h1>
 		</div>
 	</div>
 
+	<div class="form-group clearfix">
+		<div class="row">
+			<div class="col-md-4 col-sm-4 col-xs-6">
+				<label class="control-label" for="new-task-headers">Headers Object (JSON)</label>
+				<input class="form-control" id="new-task-headers" type="text" name="headers" value="{{task.headers}}" placeholder='{headers: {"Cookie": "foo=bar"}}'/>
```

Comment:
Should the placeholder here be just `{"Cookie": "foo=bar"}` rather than having the `headers` property in it?